### PR TITLE
Try space include file name

### PR DESCRIPTION
**Pull request by colorbox**
maybe danger comment fail backs to issue comment

### CODE REVIEW

**Comment by colorbox** [comment]
```suggestion
  pp 'hey'
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Gemfile [hunk 1]
```diff
@@ -18,7 +18,8 @@ gem 'uglifier', '>= 1.3.0'
 
 gem 'm'
 
-gem 'danger'
+gem 'danger', git:'https://github.com/colorbox/danger.git', branch: 'add-pp'
```
Comment (colorbox) [inline Gemfile]:
```suggestion
gem 'danger', git: 'https://github.com/colorbox/danger.git', branch: 'add-pp'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by colorbox** [comment]
```suggestion
  pp 'hey'
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by colorbox** [comment]
```suggestion
  pp 'hey'
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .gitignore [hunk 2]
```diff
@@ -29,3 +29,5 @@
 
 # Ignore master key for decrypting credentials and more.
```
Comment (colorbox) [inline .gitignore]: test

File: app/models/space_include_ _file_name.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Hoge
+  pp "hey"
```
Comment (francisfuzz) [inline app/models/space_include_ _file_name.rb]: test

Comment (colorbox) [inline app/models/space_include_ _file_name.rb:2]: test

**Comment by colorbox** [comment]
This problem has been solved without any code editing
https://github.community/t5/GitHub-API-Development-and/How-can-I-create-PR-comment-on-the-file-that-includes-space/m-p/16503

**Pull request closed without merging**